plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx

```diff
@@ -167,7 +166,6 @@ export const useTechDocsReaderDom = (
   const postRender = useCallback(
     async (transformedElement: Element) =>
       transformer(transformedElement, [
-        scrollIntoAnchor(),
```

Comment:
The Transformers only run after adding TechDocs shadow root to the dom. This means that expected navigation behavior, backwards and forwards browser arrows, will not trigger this logic to run.